i18n/fr.yaml

```diff
@@ -135,9 +135,39 @@ rss-feed-url:
 skip-to-content:
   other: "Passer au contenu principal"
 terms-and-conditions:
-  other: "Avis"
+  other: "Modalités"
+resources:
+  other: "Ressources"
+accessibility:
+  other: "Accessibilité"
+past-projects:
+  other: "Projets antérieurs"
+meet-the-team:
+  other: "Rencontrez l’équipe"
+our-values:
+  other: "Nos valeurs"
+contact-us-footer:
+  other: "Communiquez avec nous"
+newsletter:
+  other: "Infolettre"
+roadmap:
+  other: "Feuille de route vers 2025"
+beginning-the-conversations:
+  other: "Lancer la conversation"  
 terms-link:
   other: "/transparence/avis/"
+accessibility-link:
+  other: "/a11y/"
```

Comment:
This should be `/fr/a11y` I believe to go to the French page.

Reply:
Just this by itself takes you to the FR ally page. `/fr/ally` takes you to a 404 page

Reply:
You want `11` not `ll`, heh. https://digital-cana-chore-rede-yhrmjm.herokuapp.com/fr/a11y/. If I switch to French, then click the accessibility link I end up on the English page.